### PR TITLE
Update smalltalk.html.markdown

### PR DESCRIPTION
**Pull request by horrido**
- [ ] I solemnly swear that this is all original content of which I am the original author
- [ ] Pull request title is prepended with `[language/lang-code]`
- [ ] Pull request touches only one file (or a set of logically related files with similar changes made)
- [ ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: smalltalk.html.markdown [hunk 1]
```diff
@@ -30,6 +30,7 @@ The most basic operation is to send a message to an object
 `anObject aMessage`
 
 There are three sorts of messages
+
```
Comment (mxschumacher) [inline smalltalk.html.markdown]: a single newline char fixes the issues you have raised in #3832 ?

**Comment by horrido** [comment]
I hope so. I don't see what else is causing the formatting issue. It's just
a guess.

File: smalltalk.html.markdown (hunk 1 above)
Comment (divayprakash) [reply smalltalk.html.markdown]: Hey Max!

I've checked it over and it does fix the issue raised in #3829

**Comment by divayprakash** [comment]
@mxschumacher I'll leave it up to you to merge this in.

**Comment by mxschumacher** [comment]
Okay, the change is fine with me - thanks divay

**Pull request merged**